docker-compose.yml

```diff
@@ -1,7 +1,6 @@
 # Shared variables using Docker x- extension and YAML merging syntax.
 
 x-web-variables: &web-variables
-  ? ALIGNMENT_CONFIG_DEFAULT_NAME=2020-02-10
```

Comment:
I think you actually still need the `? ALIGNMENT_CONFIG_DEFAULT_NAME` part in order for the variable to get into the container

Reply:
You need it to import the environment variable from the outside environment but this one is loaded in the image itself. The command calls `chamber exec` so they are loaded by the image at startup. See:

`Dockerfile:61`

```Dockerfile
ENTRYPOINT ["bin/entrypoint.sh"]
```

`bin/entrypoint:7`

```bash
if [ "$OFFLINE" = "1" ]
then
    exec bundle exec "$@"
else
    # Use Chamber to inject secrets via environment variables.
    exec chamber exec idseq-$ENVIRONMENT-web -- bundle exec "$@"
fi
```

Reply:
Oh I see yeah